browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -1790,6 +1798,54 @@ IN_PROC_BROWSER_TEST_F(CosmeticFilteringFlagDisabledTest,
   ASSERT_EQ(true, EvalJs(contents, "checkSelector('.ad', 'display', 'block')"));
 }
 
+#if BUILDFLAG(ENABLE_PLAYLIST)
+
+namespace playlist {
+
+class CosmeticFilteringPlaylistFlagEnabledTest : public AdBlockServiceTest {
+ public:
+  CosmeticFilteringPlaylistFlagEnabledTest() {
+    feature_list_.InitAndEnableFeature(features::kPlaylist);
+  }
+
+ private:
+  base::test::ScopedFeatureList feature_list_;
+};
+
+// Ensure cosmetic filtering occurs always when AllowCosmeticFiltering() is
```

Comment:
Could you please update the comment to show the sense of what we want to test here?